.github/workflows/build.yml

```
@@ -10,29 +10,43 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Install node
-      uses: actions/setup-node@v1
-      with:
-       node-version: '10.x'
-    - name: Install Python
-      uses: actions/setup-python@v2
+
+    - uses: actions/checkout@v2
+
+    - name: Setup conda
+      uses: goanpeca/setup-miniconda@v1
       with:
-        python-version: '3.7'
-        architecture: 'x64'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install jupyterlab
+        mamba-version: "*"
+        channels: conda-forge/label/jupyterlab_rc, conda-forge/label/jupyterlab_server_rc, conda-forge/label/jupyterlab_widgets_rc, conda-forge
```

We should now be able to drop the prerelease labels:

```suggestion
        channels: conda-forge
```